docs/pages/admin-guides/management/guides/migrating-iam-ic-from-okta-to-teleport.mdx

```diff
@@ -0,0 +1,211 @@
+
+### Midway
+
+This is the point where you can start migrating control of users and groups from
+Okta to Teleport. At this stage:
+
+ - Okta provides SSO login for Identity Center
+ - Okta manages a subset of the Identity Center group membership (selected via Push Groups)
+ - Okta and Teleport both control user provisioning via SCIM
+ - Teleport manages a second subset of Identity Center groups (selected via group filters during plugin installation)
+ - Teleport controls IC Group Account Assignments for the Identity Center groups under its control
+ - Teleport controls direct IC User Account Assignments for the Identity Center users under its control
+
+![Migration Mid Point](../../../../img/identity-center/ic-migrate-mid.png)
+
+### Ending point
+
+Once Teleport is in control of all the users and groups you want to provision
+into Identity Center, the Okta Provisioning and Push Groups functions can be
+retired, leaving Okta only providing SSO login.
+
+![Migration End Point](../../../../img/identity-center/ic-migrate-end.png)
+
+## Step 1: Install Okta SAML connector
+
+Install Okta SAML connector into Teleport as per the Teleport 
+[Okta as an SSO provider](../../access-controls/sso/okta.mdx) guide.
+
+<Admonition type="note">
+ Ideally, both Identity Center and Teleport will use the same Okta SAML application
+ for their signin, which guarantees that both Okta and Teleport will view the same 
+ user set. 
+
+ If this is not possible, the most flexible approach we've found is having an
+ an Okta Group for those users that should have IC access, and then assigning
+ both the Okta Identity Center App and the Okta SAML App used for Teleport SAML
+ connector to that group. This ensures that the same user set is visible across
+ both applications.
+</Admonition>
+
+You will need the Teleport SAML Connector Name and Okta SAML App ID in the next
+step.
+
+## Step 2: Install the Teleport Okta integration
+
+We will be using a very limited subset of the Teleport Okta integration in this 
+deployment, disabling all features except periodic user synchronization. This
+configuration is not supported by the normal installation UI, so we will have to
+use `tctl` to install it:
+
+```console
+$ ./tctl plugins install okta \
+    --org ${OKTA_ORG_URL} \
+    --saml-connector ${TELEPORT_SAML_CONNECTOR_NAME} \
+    --app-id ${OKTA_SAML_APP_ID} \
+    --api-token ${OKTA_API_TOKEN} \
+    --no-scim \
+    --no-accesslist-sync \
+    --no-appgroup-sync
+```
+
+This will install the Okta integration and start the User sync service. You can
+monitor the state of the Okta integration in the Teleport Integrations UI.
+
+## Step 3: Wait for user sync
+
+To make sure everything is working, wait until the first Okta to Teleport user
+sync has occurred. You can verify this by either
+ - refreshing the user page and finding your Okta users,
+ - checking the Okta integration status page, or
+ - watching the Teleport logs
```

Comment:
Nit: I'd remove this one in favor of the previous two more user-friendly options. Plus, it won't suit Cloud customers.
```suggestion
```